content/cpp/concepts/math-functions/terms/fmax/fmax.md

```diff
@@ -0,0 +1,63 @@
+---
+Title: 'fmax()'
+Description: 'Returns the larger of the two input arguments.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **fmax()** function returns the larger of the two input arguments.
+
+## Syntax
+
+```pseudo
+fmax(x, y)
+```
+
+If one of the parameters is `NaN`, then the other parameter is returned.
```

Comment:
```suggestion
The parameters `x` and `y` are usually a numeric [data type](https://www.codecademy.com/resources/docs/cpp/data-types). If one of the parameters is `NAN`, then the other parameter is returned.
```

We should also include something here about how this functions with pos/neg `INFINITY`.

Reply:
Would something like the following work?

```text
Any numeric value is larger than negative `INFINITY` except negative `INFINITY` and `NAN`. Any numeric value is smaller than positive `INFINITY` except `INFINITY` and `NAN`.
```

As an alternative, and _before_ mentioning the specific instance where one of the parameter is `NAN`, I could add the following:

```text
Values are ordered from negative `INFINITY` to positive `INFINITY`.
```

In this manner one understands that `fmax` returns negative `INFINITY` if the other value is `NAN`.

Let me know if you have a preference or a better approach.

Reply:
@borntofrappe What about something like this, instead?

```suggestion
The parameters `x` and `y` are usually a numeric [data type](https://www.codecademy.com/resources/docs/cpp/data-types) ranging from `-INFINITY` (negative) to `INFINITY` (positive). If one of the parameters is `NAN`, then the other parameter is returned.
```

I separated "negative" and "positive" into parentheses to preserve the C++ syntax of the two infinity values.

Reply:
Excellent. I'll commit the change promptly.